### PR TITLE
 change in user profile buttons

### PR DESCRIPTION
**Pull request by SpEcHiDe**

### CODE REVIEW

**Comment by Kedarnath101** [comment]
sending a Button of KeyboardButtonUserProfile changes to `KeyboardButton` with the `text provided.`
while on Sending `InputKeyboardButtonUserProfile`, It changes to `KeyboardButtonUserProfile.`

So, actually they are not type of button you can send, Its is given as output/return type of Input buttons. 
Also bot api docs mention only of inline keyboard button of user mentions

**Review by Lonami — CHANGES_REQUESTED** [review]
I think it would be a better idea to provide a new `.inline_mention` method. `.mention(True, 'text')` seems a bit strange.

**Comment by Lonami** [comment]
@SpEcHiDe what do you think about my suggested approach https://github.com/LonamiWebs/Telethon/pull/3680#pullrequestreview-863276502?

**Comment by SpEcHiDe** [comment]
@Lonami can you review once more, please?

File: telethon/types/_custom/button.py [hunk 1]
```diff
@@ -185,10 +196,22 @@ def mention(text, input_entity=None):
                     For now, you cannot use ID or username for this argument.
                     If you want to use different user, you must manually use
                     `client.get_input_entity() <telethon.client.users.UserMethods.get_input_entity>`.
+
+             is_inline:
+                 whether to create inline / keyboard buttons
```
[the remark this answers is not in the document]
Comment (SpEcHiDe) [reply telethon/types/_custom/button.py]: sorry, again.

Can you check once more?

**Pull request merged**